docs/writing-tests/test-runner.md

```diff
@@ -0,0 +1,301 @@
+---
+title: 'Test runner'
+---
+
+Storybook test runner turns all of your stories into executable tests. It is powered by [Jest](https://jestjs.io/) and [Playwright](https://playwright.dev/).
```

Comment:
You might want to double check every place in this PR that mentions this. Some places only mention Playwright, others mention Jest + Playwright

Reply:
I've updated the docs accordingly! There was a scenario that required clarity on this. The rest is left as is to focus the documentation as mentioning either can be misread or interpreted incorrectly.